components/learn/modules/ROOT/pages/deploy-and-interact.adoc

```diff
@@ -58,7 +58,7 @@ $ npx openzeppelin init
 
 NOTE: You can call the OpenZeppelin CLI with both `openzeppelin` and `oz`. For brevity, from here on we'll use the latter.
 
-Two things will happen during initialization. First, an `.openzeppelin` directory will be created, holding project-specific information. This directory will be managed by the CLI: you won't need to manually edit anything. It is important however to commit any changes here to your Git repository.
+Two things will happen during initialization. First, an `.openzeppelin` directory will be created, holding project-specific information. This directory will be managed by the CLI: you won't need to manually edit anything. It is important however to commit any changes here to your Git repository (local network files like `dev-<network_id>.json` and the `.session` file https://docs.openzeppelin.com/sdk/2.6/configuration#configuration-files-in-version-control[should not be tracked in Git]).
```

Comment:
```suggestion
Two things will happen during initialization. First, an `.openzeppelin` directory will be created, holding project-specific information. This directory will be managed by the CLI: you won't need to manually edit anything. You should however commit xref:cli::configuration.adoc#configuration-files-in-version-control[some of these files] to Git.
```

Having https://github.com/OpenZeppelin/openzeppelin-sdk/issues/363 fixed would let us simplify this.

Reply:
Updated text

Reply:
Updated